incubator/fluentd/values.yaml

```diff
@@ -33,13 +32,16 @@ papertrail:
   enabled: false
```

Comment:
Can the image.tag be updated here too (line 29), even though this is commented out? Would be good for consistency, to imply that the new version is the new minimum everywhere.